include/robot_localization/navsat_transform.hpp

```diff
@@ -32,15 +32,18 @@
 #ifndef ROBOT_LOCALIZATION__NAVSAT_TRANSFORM_HPP_
 #define ROBOT_LOCALIZATION__NAVSAT_TRANSFORM_HPP_
 
-#include <memory>
+#include <stdexcept>
```

Comment:
Nit: these were alphabetized within the groups (in this case, STL headers). Would you mind retaining that?